app/src/main/java/org/mozilla/focus/ext/Context.kt

```diff
@@ -17,3 +18,11 @@ val Context.appVersionName: String?
         val packageInfo = packageManager.getPackageInfo(packageName, 0)
         return packageInfo.versionName
     }
+
+val Context.isVoiceViewEnabled: Boolean
+    get() {
+        // In my understanding, isTouchExplorationEnabled should represent VoiceView being enabled,
+        // whereas isEnabled says if the accessibility manager is enabled for other services.
+        val am = this.getSystemService(Context.ACCESSIBILITY_SERVICE) as? AccessibilityManager ?: return false
```

Comment:
I'm assuming that we wouldn't need to listen for if the accessibility changes - you probably can't do that in the middle of the app running.